ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/processor/TransactionTrace.java

```diff
@@ -28,12 +30,14 @@ public class TransactionTrace {
   private final TransactionProcessingResult result;
   private final List<TraceFrame> traceFrames;
   private final Optional<Block> block;
+  private final Optional<Collection<AccessLocationTracker.AccountAccessList>> touchedAccounts;
```

Comment:
Is there any need to distinguish between an empty list and a empty optional? could the type just be `Collection<AccessLocationTracker.AccountAccessList>` instead?